enclave/src/integration-test/java/tech/pegasys/pantheon/enclave/EnclaveTest.java

```diff
@@ -80,6 +82,18 @@ public void testSendAndReceive() throws Exception {
     assertNotNull(rr.getPrivacyGroupId());
   }
 
+  @Test
+  public void testCreatePrivacyGroup() throws Exception {
+    List<String> publicKeys = testHarness.getPublicKeys();
+    CreatePrivacyGroupRequest privacyGroupRequest =
+        new CreatePrivacyGroupRequest(
+            publicKeys.toArray(new String[0]), publicKeys.get(0), "test", "test");
```

Comment:
consider using different values like "testName" & "testDescription" so that we can verify everything got stored in the correct place

Reply:
I question the value of this test as it doesn't actually confirm that anything was done correctly.  Can we confirm the data out of the enclave directly?  is findPrivacyGroups implemented yet?

Reply:
What are the expected failure cases?  Can those be tested?

Reply:
We dont have a findPrivacyGroups API yet. Once thats implemented, we can test the correctness of privacy group Id.

Reply:
OK, I updated that issue to include coming back to this.